Assets/HoloToolkit/Input/Scripts/Utilities/Managers/MixedRealityCameraManager.cs

```diff
@@ -3,12 +3,8 @@
 
 using UnityEngine;
 
-#if UNITY_WSA
-#if UNITY_2017_2_OR_NEWER
+#if UNITY_WSA && UNITY_2017_2_OR_NEWER
```

Comment:
Didn't mean to include the Remoting fix, but it should probably be in the release anyway.